src/ui/public/utils/monday_first_weekdays.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+
+/*
+ * ElasticSearch dayOfWeek lucene expression always returns Monday as the first day of week.
+ * Depending on the locale, Moment will return different days as the first day of week.
+ * This method creates a weekdays array with locale spellings but
+ * where Monday is the first day regardless of locale settings.
+ */
+export default function mondayFirstWeekdays() {
```

Comment:
This seems kind of weird as a function, the values never change. If you're worried about mutation, maybe just `Object.freeze` the value?